library/restoreSession.php

```diff
@@ -28,7 +29,15 @@
 var oemr_session_name = <?php echo json_encode(urlencode(session_name())); ?>;
 var oemr_session_id   = <?php echo json_encode(urlencode(session_id())); ?>;
 var oemr_dialog_close_msg = <?php echo (function_exists('xlj')) ? xlj("OK to close this other popup window?") : json_encode("OK to close this other popup window?"); ?>;
-//
+
+var oemr_scp_lifetime = <?php echo $scparams['lifetime']; ?>; // must be numeric
```

Comment:
note json_encode/js_escape should keep it numeric if needed (pretty sure it does not place quotes around numbers)